examples/wind/src/layers/particle-layer/particle-layer.js

```diff
@@ -5,20 +5,19 @@ import {
   Model,
   Geometry,
   Buffer,
-  TransformFeedback,
   setParameters,
   loadTextures,
   Texture2D,
-  Program
+  experimental
 } from 'luma.gl';
 
 import {ELEVATION_DATA_IMAGE, ELEVATION_DATA_BOUNDS, ELEVATION_RANGE} from '../../defaults';
 
 import vertex from './particle-layer-vertex.glsl';
 import fragment from './particle-layer-fragment.glsl';
 import vertexTF from './transform-feedback-vertex.glsl';
-import fragmentTF from './transform-feedback-fragment.glsl';
 
+const {Transform} = experimental;
```

Comment:
Nit: I'd move this line to just under the luma.gl import. It is "in spirit" a luma.gl import;